jms/src/main/mima-filters/1.0.2.backwards.excludes

```diff
@@ -8,3 +8,10 @@ ProblemFilters.exclude[MissingTypesProblem]("akka.stream.alpakka.jms.DurableTopi
 
 # add Java API
 ProblemFilters.exclude[ReversedMissingMethodProblem]("akka.stream.alpakka.jms.JmsEnvelope.*")
+
+# PR #1830
+ProblemFilters.exclude[ReversedMissingMethodProblem]("akka.stream.alpakka.jms.JmsMessage.withProperty")
+ProblemFilters.exclude[ReversedMissingMethodProblem]("akka.stream.alpakka.jms.JmsMessage.withHeaders")
+ProblemFilters.exclude[ReversedMissingMethodProblem]("akka.stream.alpakka.jms.JmsMessage.withProperties")
+ProblemFilters.exclude[ReversedMissingMethodProblem]("akka.stream.alpakka.jms.JmsMessage.withHeader")
+ProblemFilters.exclude[ReversedMissingMethodProblem]("akka.stream.alpakka.jms.JmsMessage.withoutDestination")
```

Comment:
These have to go to a separate file named by the latest released, so `1.1.0.backwards.excludes`